pkgs/applications/networking/instant-messengers/riot/riot-desktop.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub, yarn2nix, makeDesktopItem, electron, riot-web }:
+
+let
+  executableName = "riot-desktop";
+  version = "1.0.4";
```

Comment:
Don't see why to `let in` this.

Reply:
It's used both at lines 9 and 15 so that the version number stays consistent.